src/views/PaymentNetwork.vue

```diff
@@ -74,54 +74,72 @@ function formatFiatCurrency(amount: number): string {
   return `${BASE_CURRENCY.symbol}${Math.round(amount).toLocaleString()}`;
 }
 
-function formatCryptoCurrency(amount: number, currency): string {
-  return `${amount.toLocaleString()} ${currency.code}`;
-}
-
 watch(paymentTx, () => {
   modalPostPaymentOpen.value = !!paymentTx.value;
 });
 </script>
 
 <template>
-  <h1>Network plan</h1>
-  <p class="text-md mb-4 leading-5">Pay to support your network on Snapshot</p>
+  <h1 class="leading-[44px] mb-[2px]">Network plan</h1>
+  <p class="mb-4 leading-[20px]">Pay to support your network on Snapshot</p>
 
   <form v-if="fxLoadStatus === 1" class="flex flex-col gap-4" @submit="pay">
     <fieldset>
-      <legend>
-        <h2 class="text-lg mb-2">Select your plan duration</h2>
+      <legend
+        class="text-[20px] mb-[12px] leading-none text-skin-heading font-semibold"
+      >
+        Select your plan duration
       </legend>
       <ol class="flex flex-col gap-2">
-        <li v-for="(plan, planId) in PLANS" :key="planId">
+        <li
+          v-for="(plan, planId) in PLANS"
+          :key="planId"
+          :class="[
+            {
+              'bg-[#384aff]/20 rounded-xl px-[1px] pb-[1px] mb-[-1px]':
+                plan.discount
+            }
+          ]"
+        >
+          <div v-if="plan.discount" class="p-2 text-center text-skin-primary">
+            Save
+            <BasePill class="py-1 !bg-skin-primary"
+              >{{ Math.floor(plan.discount) }}%</BasePill
+            >
+          </div>
           <label
-            class="flex gap-3 border border-skin-border hover:border-skin-text rounded-xl p-3 cursor-pointer"
+            class="flex relative bg-skin-bg gap-3 ring-1 ring-skin-border hover:ring-skin-text rounded-xl py-[12px] px-3 cursor-pointer"
             :class="{
-              '!border-skin-primary': planId === data.plan
+              '!ring-skin-primary': planId === data.plan,
+              'ring-[#384aff]/0': plan.discount
             }"
             @click="setData('plan', planId)"
           >
-            <div class="flex-grow">
-              <b class="text-skin-heading">{{ plan.label }}</b>
-              <BasePill v-if="plan.discount" class="ml-2 py-1 !bg-skin-primary"
-                >-{{ Math.floor(plan.discount) }}%</BasePill
-              >
-              <small class="block text-skin-text">
-                {{ formatFiatCurrency(computePlanFiatPrice(plan, BASE_UNIT)) }}
-                per month
-              </small>
+            <div class="flex-grow leading-none gap-[2px] flex flex-col">
```

Comment:
What does leading-none do here?

Reply:
Removing the default line-height

Reply:
Should be one line below I think? Two lines below you add `leading-none` again